scoresheets/SmoothieChef.tex

```diff
@@ -0,0 +1,14 @@
+The maximum time for this test is 15 minutes.
+
+\begin{scorelist}
+	\scoreheading{Main Goal}
+	\scoreitem[5]{500}{Place fruit items into the bowl}
```

Comment:
This seems like a LOT. Object rec on the bowl after demonstration then one pick and place operation in a fairly controlled environment (compared to let's say storing groceries) gives as much points as a full Stage 1 tasks.